src/lib/utils/getTime.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * If val is a date, return its time. Otherwise, return val.
+ */
+export default function getTime (val) {
+	if (Object.prototype.toString.call(val) === '[object Date]') {
```

Comment:
Mostly curious, could this be `if (val instanceof Date) {` instead?  Maybe this handles some additional use cases.

Reply:
I was reading about the differences and it seems like the `instanceof` approach doesn't work in certain scenarios where the date was created by another frame. It seems like an obscure use case but I look at what underscore does and they use this approach.

Reply:
Thanks.  I figure there was some nuance (always is with javascript :) ).